.circleci/scripts/check_app_status.py

```diff
@@ -14,15 +14,15 @@ def check_app_status():
                             capture_output=True,
```

Comment:
I started editing this file because my build failed. It turned out to be most likely a race condition, so i didnt change the functionality but am leaving the linting.

Reply:
👍